File: allSpiceButTheGoodOne/Models/Recipe.cs
namespace allSpiceButTheGoodOne.Models
{
    public class Recipe
    {
        public int Id { get; set; }
        public string CreatorId { get; set; }
        public string Title { get; set; }
        public string Category { get; set; }
        public string Instructions { get; set; }
        public string Img { get; set; }
        public Profile Creator { get; set; }
    }
}

File: allSpiceButTheGoodOne.client/src/models/Recipe.js

export class Recipe {
    constructor(data) {
        this.activeRecipe = data.activeRecipe || null
        this.id = data.id
        this.creatorId = data.creatorId
        this.title = data.title
        this.category = data.category
        this.instructions = data.instructions
        this.img = data.img
    }
}